site/website/blog/2020-10-05-jet-43-is-released.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Jet 4.3 is Released
+author: Marko Topolnik
+authorURL: https://twitter.com/mtopolnik
+authorImageURL: https://pbs.twimg.com/profile_images/922726943614783488/Pb5DDGWF_400x400.jpg
+---
+
+Today we're releasing Hazelcast Jet 4.3, our fourth release in 2020!
+
+We took part in Google Summer of Code that ended just a few weeks ago,
+and this release already brings a productionized piece of work that our
+student, [Mohamed Mandouh](https://github.com/MohamedMandouh), made:
+distributed in-memory sorting. Mohamed's primary focus was research into
+the feasibility of integrating RocksDB or a similar DB library as a
+disk-based state backend for Jet's hash join, aggregation and sorting,
+and we plan to continue with this work for some more time.
```

Comment:
Wasn't there some kind of show stopper mentioned related to this topic? Will we really continue in the near future?

Reply:
There's the issue of blocking on cooperative threads to be solved, a possible solution is a dedicated filesystem IO thread that concentrates all blocking ops that go to the disk.